src/main/java/org/openhab/tools/analysis/utils/LineFormatterCallback.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.tools.analysis.utils;
+
+import org.openhab.tools.analysis.checkstyle.readme.MarkdownCheck;
+
+/**
+ * A callback interface for {@link MarkdownCheck}
+ *
+ * @author Lyubomir Papazov - initial contribution
+ *
+ */
+public interface LineFormatterCallback {
+    /**
+     * Used in AbstractStaticCheck's findLineNumber method to apply rules
+     * about escaping special characters to a line in the .mardown file
```

Comment:
typo .mardown -> markdown 

But maybe it is better .md as this is the file extension that we are using in the check, or just say from the markdown file (without the .)